src/data_processing/netflix_to_wikidata.py

```diff
@@ -39,7 +39,7 @@ def construct_query(title, year):
         ?item wdt:P31/wdt:P279* wd:Q11424 .
 
         ?item wdt:P577 ?releaseDate .
-        FILTER (YEAR(?releaseDate) >= %d && YEAR(?releaseDate) <= %d) .
+        FILTER (YEAR(?releaseDate) >= %(start_year)d && YEAR(?releaseDate) <= %(end_year)d) .
```

Comment:
I just tested this in at query.wikimedia.org: https://w.wiki/B4XX

The only problem is that if you use `&& YEAR(?releaseDate <= 2003`, it captures any movie that was released in 2003 (https://w.wiki/B4XX). It looks like you can just use `FILTER (YEAR(?releaseDate) = 2002)` and skip the ranges.

Reply:
What about the differences in release date between the dvd and theater release tho? Are we cool with missing those if the dvd version was released in a different year?

Reply:
Actually as I put below, it only misses out on 2/100 movies, so updated